### DISCUSSION TITLE
archives/Gamer/%E6%98%9F%E9%93%81%E9%AB%98%E7%AD%89%E9%80%9F%E5%BA%A6%E8%AE%BA%E5%9F%BA%E7%A1%80/

### DISCUSSION THREAD

**Discussion by giscus** [4rozeN/4rozeN.github.io#6] [General]
# archives/Gamer/%E6%98%9F%E9%93%81%E9%AB%98%E7%AD%89%E9%80%9F%E5%BA%A6%E8%AE%BA%E5%9F%BA%E7%A1%80/

星铁高等速度论基础 本文旨在介绍星铁的速度机制和探讨0T或追0T的排轴计算。  前置知识 ​	星铁首轮行动值为150，往后的每轮为100。 ​	角色一动需要跑完10000(1万)的路程，两动就是20000(2万)，以此类推。  行动值 星铁的行动值需要在设置中打开。 你可以在ESC设置——其他——战斗功能——战斗中是否显示角色行动值这一选项中将其开启。  什么是行动值？或者说星铁的行动值是

http://localhost:4000/archives/Gamer/%E6%98%9F%E9%93%81%E9%AB%98%E7%AD%89%E9%80%9F%E5%BA%A6%E8%AE%BA%E5%9F%BA%E7%A1%80/

**Comment by 4rozeN** [comment]
test.
